bin/tass

```diff
@@ -109,6 +109,19 @@ when 'create'
         end
       end
     end
+
+    if config.key?(:asg_metrics)
+      # We list all metrics because otherwise we can't get GroupStandbyInstances
+      Tapjoy::AutoscalingBootstrap::AWS::Autoscaling.client.enable_metrics_collection({
+        auto_scaling_group_name: Tapjoy::AutoscalingBootstrap.scaler_name,
+        metrics: %w(
```

Comment:
I believe you you can pass metrics an empty array and it will default to providing all available metrics. Toss up here as being explicit tells us exactly what metrics we will have, but may mean we need to add new metrics as they become available. Not likely to change a lot, so this is fine as is.